src/pages/Expense.tsx

```diff
@@ -39,29 +42,42 @@ const Expense: React.FC<Props> = (props) => {
 
   function componentForPerson(name: string, idx: number) {
     const shares = sharesPerPerson.get(idx) ?? 0;
-    let part;
-    if (totalShares > 0 && price.isPositive()) {
-      part = (
-        <IonLabel>
-          ~ ${price.multiply(shares).divide(totalShares).toUnit()}
-        </IonLabel>
-      );
-    }
+    const part = (
+      <IonLabel>
+        ~ $
+        {totalShares > 0 && price.isPositive()
+          ? price.multiply(shares).divide(totalShares).toUnit()
+          : Dinero().toUnit()}
+      </IonLabel>
+    );
+
+    const changeShares = (delta: number) => {
+      const newShares = new Map(sharesPerPerson);
+      newShares.set(idx, shares + delta);
+      setSharesPerPerson(newShares);
+    };
     return (
       <IonItem key={idx}>
-        <IonLabel>{name}:</IonLabel>
-        <IonInput
-          type="number"
-          inputmode="numeric"
-          placeholder="0"
-          value={shares}
-          onIonChange={(e) => {
-            const newShares = new Map(sharesPerPerson);
-            newShares.set(idx, parseToFloat(e.detail.value));
-            setSharesPerPerson(newShares);
-          }}
-        ></IonInput>
-        {part}
+        <IonGrid>
+          <IonRow>
+            <IonCol size="4">{name}:</IonCol>
+            <IonCol>
+              <IonButton
+                onClick={() => changeShares(-1)}
+                disabled={shares === 0}
+              >
+                -
+              </IonButton>
+            </IonCol>
+            <IonCol>
+              <IonLabel className="ion-text-center">{shares}</IonLabel>
```

Comment:
Is it possible to still leave this as an input? And then you can edit with the buttons but also writing it down? I see some apps that do this.